src/main/java/org/veupathdb/lib/container/jaxrs/repo/SQL.java

```diff
@@ -39,7 +42,33 @@
         {
           private static final String table = Tables.UserDB.UserSchema.Users;
 
-          static final String GuestByID = select(db, schema, table, "guest-by-id");
```

Comment:
Converted to a lazy load value since the static instantiation would have happened before the options were loaded from CLI.